physics/src/modules/IceThermodynamicsModule/ThermoIce0.cpp

```diff
@@ -87,6 +87,7 @@ void ThermoIce0::calculateElement(size_t i, const TimestepTime& tst)
     if (hice[i] == 0. || cice[i] == 0.) {
         deltaHi[i] = 0.;
         snowToIce[i] = 0.;
+        tice.zIndexAndLayer(i, 0) = freezingPointIce;
```

Comment:
This should be in a bug fix PR, probably along with `BasicIceOceanHeatFlux:30`.

Reply:
Now in #789